src/components/common/Header.js

```diff
@@ -129,74 +90,128 @@ const useStyles = makeStyles(() => ({
       color: listSelectedTextColor,
     },
   },
-}))
+}));
+
+const breadcrumbs = [
+  { id: 'home', text: 'Home' },
+  { id: 'breadSubItem1', text: 'breadSubItem1' },
+  { id: 'breadSubItem2', text: 'breadSubItem2' },
+  { id: 'breadSubItem3', text: 'breadSubItem3' },
+  { id: 'breadSubItem4', text: 'breadSubItem4' },
+  { id: 'breadSubItem5', text: 'breadSubItem5' },
+  { id: 'breadSubItem6', text: 'breadSubItem6' },
+  { id: 'breadSubItem7', text: 'breadSubItem7' },
+  { id: 'composition2', text: 'Composition 2' },
+];
+
+const listItems = [
+  { label: 'Build', soon: false },
+  { label: 'Test', soon: true },
+  { label: 'Composition', soon: false },
+];
 
 const Header = () => {
   const classes = useStyles();
   const [anchorEl, setAnchorEl] = React.useState(null);
+  const [selected, setSelected] = React.useState('Build');
   const open = Boolean(anchorEl);
 
-  const handleClick = (event) => {
-    setAnchorEl(anchorEl ? null : event.currentTarget);
+  const handleClick = (event, value) => {
+    if (event) {
+      if (value === 'Composition') {
```

Comment:
same here, try avoiding hard coded values in if then constructs use constants instead of strings